Assets/GameAssets/Boss/BossScripts/Attacks/BaseBossAttack.cs

```diff
@@ -48,6 +53,40 @@ protected virtual void BeginAttack()
         _isAttackActive = true;
     }
 
+    protected virtual void PlayIdleLoop(float second)
+    {
+        if (_idleSFXCoroutine == null)
```

Comment:
Is this supposed to return if null? There is no return

Reply:
not really just for starting the coroutine but I can add that